pkg/cache/v2/cache.go

```diff
@@ -94,7 +124,29 @@ type RawResponse struct {
 	marshaledResponse atomic.Value
 }
 
+// RawDeltaResponse is a pre-serialized xDS response that utilizes the delta discovery request/response objects.
+type RawDeltaResponse struct {
+	// Request is the original request.
+	DeltaRequest *discovery.DeltaDiscoveryRequest
+
+	// System Version Info
+	SystemVersionInfo string
+
+	// Resources to be included in the response.
+	Resources []types.Resource
+
+	// RemovedResources is a list of unsubscribed aliases to be included in the response
```

Comment:
Is this not the list of resources that should be removed on the client? I don't think this affects subscriptions

Reply:
Yes sorry that comment is misleading. Will change